server/group_test.go

```diff
@@ -121,7 +121,22 @@ func TestApplicationHandler_CreateGroup(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	bodyReader := strings.NewReader(`{"name": "ABC_DEF_TEST_UPDATE"}`)
+	bodyReader := strings.NewReader(`{
```

Comment:
@Dotunj There should be more tests for this guy. Like catch validation errors using a test table.